Build problem with IFPEN gimkl2021b toolchain

Hello
Build failed with:
[  1%] Building CXX object femutils/CMakeFiles/FemUtils.dir/HypreDoFLinearSystem.cc.o
/work/guignont/hackathon_cines/arcanefem/femutils/HypreDoFLinearSystem.cc: Dans la fonction membre « virtual void Arcane::FemUtils::HypreDoFLinearSystemImpl::solve() »:
/work/guignont/hackathon_cines/arcanefem/femutils/HypreDoFLinearSystem.cc:289:5: erreur: « HYPRE_SetSpGemmUseVendor » n'a pas été déclaré dans cette portée; vouliez-vous employer « HYPRE_SetSpGemmUseCusparse » ?
  289 |     HYPRE_SetSpGemmUseVendor(false);
      |     ^~~~~~~~~~~~~~~~~~~~~~~~
      |     HYPRE_SetSpGemmUseCusparse
make[2]: *** [femutils/CMakeFiles/FemUtils.dir/build.make:186: femutils/CMakeFiles/FemUtils.dir/HypreDoFLinearSystem.cc.o] Error 1

module  Hypre/2.23.0 is loaded.

--- a/femutils/HypreDoFLinearSystem.cc
+++ b/femutils/HypreDoFLinearSystem.cc
@@ -284,7 +284,7 @@ solve()
   hypreCheck("HYPRE_SetExecutionPolicy", HYPRE_SetExecutionPolicy(hypre_exec_policy));
 
   if (is_use_device) {
-#if HYPRE_RELEASE_NUMBER >= 22300
+#if HYPRE_RELEASE_NUMBER >= 22700
     /* use hypre's SpGEMM instead of vendor implementation */
     HYPRE_SetSpGemmUseVendor(false);
 #endif

